src/main/java/edu/kis/powp/command/FigureFactory.java

```diff
@@ -0,0 +1,19 @@
+package edu.kis.powp.command;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+public class FigureFactory {
+
+    public ComplexCommand makeSquare(Job2dDriver driver){
```

Comment:
DriverCommand return type would be sufficient.
This factory is too small and there is no test verifying how everything works. Missing point 3.1.5.